### PR TITLE
blog: announce 24.11 release

### PR DESCRIPTION
**Pull request by RossComputerGuy**
*To be merged on Saturday, November 30th*

### CODE REVIEW

**Review by thilobillerbeck — APPROVED** [review]

File: src/content/blog/announcements/2024-11-30_nixos-24.11.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+- [Release manual](/manual/nixos/stable/release-notes.html#sec-release-24.11)
+  - [Highlights](/manual/nixos/stable/release-notes.html#sec-release-24.11-highlights)
+  - [New Services](/manual/nixos/stable/release-notes.html#sec-release-24.11-new-services)
+  - [Backward Incompatibilities](/manual/nixos/stable/release-notes.html#sec-release-24.11-incompatibilities)
+  - [Other Notable Changes](/manual/nixos/stable/release-notes.html#sec-release-24.11-notable-changes)
+- [Upgrade Instructions](/manual/nixos/stable/index.html#sec-upgrading)
+- [ISO Image Downloads](/download)
+
+The 24.05 release was made possible due to the efforts of **2669** contributors, who authored **49079** commits since the previous release.
```
Comment (riotbib) [inline src/content/blog/announcements/2024-11-30_nixos-24.11.md:24]: You meant 24.11, right?

Are those stats up-to-date?

File: src/content/blog/announcements/2024-11-30_nixos-24.11.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+id: nixos-24.11
+title: NixOS 24.11 released
+date: 2024-11-30T11:00:00.000Z
+category: announcements
+---
+
+[![NixOS 24.11 Vicuña logo](../../../assets/logo/nixos-logo-24.11-vicuna-lores.png)](https://github.com/NixOS/nixos-artwork/blob/master/releases/24.05-vicuna/vicuna.png)
```
Comment (riotbib) [inline src/content/blog/announcements/2024-11-30_nixos-24.11.md:8]: `24.11-vicuna` in the path to `nixos-artwork` needs to be changed.

File: src/content/blog/announcements/2024-11-30_nixos-24.11.md (hunk 1 above)
Comment (RossComputerGuy) [reply src/content/blog/announcements/2024-11-30_nixos-24.11.md]: > You meant 24.11, right?

Yes

> Are those stats up-to-date?

Yes, I generated them yesterday. You can see they came from the NixOS release management chat on Matrix.

File: src/content/blog/announcements/2024-11-30_nixos-24.11.md (hunk 2 above)
Comment (RossComputerGuy) [reply src/content/blog/announcements/2024-11-30_nixos-24.11.md]: I fixed the path.

**Comment by thilobillerbeck** [comment]
Are we ready to merge?

**Comment by RossComputerGuy** [comment]
I'm waiting on the infra PR to merge then we can merge this.

**Pull request merged**